Write SDK maintenance policy

vcampitelli opened the pull request:
The idea behind this is to have a maintenance/deprecation policy on the SDKs / client libraries.

I've read a lot of them (e.g. [AWS'](https://docs.aws.amazon.com/sdkref/latest/guide/maint-policy.html) and [Azure's](https://azure.github.io/azure-sdk/policies_support.html)), but they are too complex for us, as they have a whole lifecycle that spans over a year, so I tried to make things simpler.

This branch contains 2 commits for two alternatives:

1. 3da532757c9201b7d79029b8ecb0347d79539dce has a simpler approach:
  ![Alternative 1 - Without timeframe](https://github.com/FusionAuth/fusionauth-site/assets/1877191/85ed9937-7b62-4351-8fe2-e08c45baff84)
2. The latest (63ee130c5357025c8ab4bed171a9444bbc2a6e98) has a time commitment of 6 months (that we can discuss):
  ![Alternative 2 - With timeframe](https://github.com/FusionAuth/fusionauth-site/assets/1877191/0014c153-d99a-4436-82bc-50a107caae6c)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206600371228594

mooreds reviewed (approved):
Let's go with the simpler approach.

Places to add it

    the SDKs page on the website
    the readmes

vcampitelli commented:
@mooreds RE: the SDKs page on the website, do you want me to add this to every SDK page? Right now, it's only on the index page (`/docs/sdks`)

mooreds commented:
I'd add it to every page, under a `### Upgrade Policy` heading and using a re-usable component.

vcampitelli commented:
@mooreds I've added the policy to every page and am about to create PRs on every SDK/Client Library. Here are the texts that I wrote and want your feedback on:

For SDKs:
> ## Upgrade Policy
>
> This library may periodically receive updates with bug fixes, security patches, tests, code samples, or documentation changes.
> These releases may also update dependencies, language engines, and operating systems, as we\'ll follow the deprecation and sunsetting policies of the underlying technologies that the libraries use.
>
> This means that after a dependency (e.g. language, framework, or operating system) is deprecated by its maintainer, this library will also be deprecated by us, and may eventually be updated to use a newer version.

For Client Libraries:
> ## Upgrade Policy
>
> This library is built automatically to keep track of the FusionAuth API, and may also receive updates with bug fixes, security patches, tests, code samples, or documentation changes.
>
> These releases may also update dependencies, language engines, and operating systems, as we\'ll follow the deprecation and sunsetting policies of the underlying technologies that it uses.
>
> This means that after a dependency (e.g. language, framework, or operating system) is deprecated by its maintainer, this library will also be deprecated by us, and will eventually be updated to use a newer version.
```

mooreds commented:
Yes, this looks great @vcampitelli

vcampitelli commented:
@mooreds I've rebased the branch, fixed a few typos and opened PRs to add the "Upgrade policy" to all of these:

- https://github.com/FusionAuth/fusionauth-example-client-libraries/pull/11
- https://github.com/FusionAuth/fusionauth-javascript-client/pull/6
- https://github.com/FusionAuth/fusionauth-netcore-client/pull/73
- https://github.com/FusionAuth/fusionauth-node-client/pull/22
- https://github.com/FusionAuth/fusionauth-php-client/pull/32
- https://github.com/FusionAuth/fusionauth-python-client/pull/23
- https://github.com/FusionAuth/fusionauth-typescript-client/pull/101
- https://github.com/FusionAuth/fusionauth-android-sdk/pull/40
- https://github.com/FusionAuth/fusionauth-angular-sdk/pull/29
- https://github.com/FusionAuth/fusionauth-javascript-sdk/pull/19
- https://github.com/FusionAuth/fusionauth-javascript-sdk-express/pull/5
- https://github.com/FusionAuth/fusionauth-react-sdk/pull/98
- https://github.com/FusionAuth/fusionauth-vue-sdk/pull/19
- https://github.com/FusionAuth/fusionauth-android-client/pull/1
- https://github.com/FusionAuth/fusionauth-elixir-client/pull/1
- https://github.com/FusionAuth/fusionauth-java-client/pull/40
- https://github.com/FusionAuth/fusionauth-swift-sdk/pull/1

mooreds commented:
okay, what do you need from me to merge these?

vcampitelli commented:
@mooreds I just need you to merge them actually, as I don't have write access on some of them
